torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -204,6 +298,13 @@ def _wrap_fx_args_as_onnxscript_args(
     complete_kwargs: Dict[str, Any] = {}
     if inspect.isbuiltin(node.target):
         complete_args = list(node.args)
+    elif isinstance(node.target, torch._ops.OpOverloadPacket):
+        # aten::sym_size is the only OverloadPacket that we support.
+        # schema: aten::sym_size(Tensor self, int dim) -> Tensor
+        assert str(node.target) == "aten.sym_size"
```

Comment:
Is this "elif" branch mainly for the assertion check?

Reply:
No. As OpOverloadPacket doesn't have schema, it would do nothing if we don't handle it specifically.

Reply:
Discussed offline, root cause #97201.

Reply:
Now it's manually pointing to torch.ops.aten.sym_int.int